tests/robust/test_internals_compositions.py

```diff
@@ -0,0 +1,59 @@
+import functools
+import warnings
+
+import pyro
+import torch
+
+from chirho.robust.internals.linearize import (
+    conjugate_gradient_solve,
+    make_empirical_fisher_vp,
+)
+from chirho.robust.internals.predictive import NMCLogPredictiveLikelihood
+from chirho.robust.internals.utils import make_functional_call
+
+from .robust_fixtures import SimpleGuide, SimpleModel
+
+pyro.settings.set(module_local_params=True)
+
+
+def test_empirical_fisher_vp_nmclikelihood_cg_composition():
```

Comment:
If this is the test you expect to fail without #408, can you mark it with `pytest.mark.xfail(reason="fails without fix in https://github.com/BasisResearch/chirho/pull/408")`?

Reply:
Given that the change in #408 now just uses `SeedMessenger` in the `linearize` body, I can fix these errors by doing the same in these two tests. I'll push up these changes now.

Reply:
That's fine, but remember to remove the `SeedMessenger` here in #408, since you want this test to exercise the changes made to `NMCLogPredictiveLikelihood` there.

Reply:
Ok in #408, I followed your suggestion here https://github.com/BasisResearch/chirho/pull/408#pullrequestreview-1760679539, and did not actually modify `NMCLogPredictiveLikelihood`. I just checked if 

```
log_prob = NMCLogPredictiveLikelihood(
        model, guide, num_samples=3, max_plate_nesting=3
    )
log_prob_params, func_log_prob = make_functional_call(log_prob)
func_log_prob = SeedMessenger(123)(func_log_prob)
```
works in the `test_nmc_likelihood_seeded` test in #408. I think we can move this discussion to #408 if you have other suggestions on this point!